### PR TITLE
sim_vehicle.py : style correction

### PR DESCRIPTION
**Pull request by khancyr**
Rework of https://github.com/ArduPilot/ardupilot/pull/4092 
Using pep8 and pylint without E501 line too long and E116 unexpected indentation (comment)

Corrections done :
Code style uniformization
Remove numerous Redefining name from outer scope  (redefined-outer-name)
Remove Arguments number differs from overridden 'parse_args' method (arguments-differ)
Rename Invalid argument name "procName" (invalid-name)
Rename Invalid variable name "outputLines" (invalid-name)
Rename Invalid variable name "lineSplit" (invalid-name)
Remove Redefinition of pid type from int to str (redefined-variable-type)

Change .has_key() (is deprecated), to use 'in'

Change Function call with set literal (at line 105)

### CODE REVIEW

**Comment by lucasdemarchi** [comment]
LGTM

File: Tools/autotest/sim_vehicle.py [hunk 1]
```diff
@@ -51,71 +55,78 @@ def parse_args(self):
...
+            except:
+                pid = int(line_split[1].strip())
+                str_pid = str(pid)
+            if str_pid not in pids:
+                pids.append(str_pid)
     return pids
 
+
 def under_cygwin():
+    """Return Cygwin path"""
```
Comment (OXINARF) [inline Tools/autotest/sim_vehicle.py]: This doesn't return the Cygwin path, it returns the existence of the cygwin binary

File: Tools/autotest/sim_vehicle.py [hunk 2]
```diff
@@ -144,206 +155,208 @@ def kill_tasks():
...
-''')
+""")
 
-parser.add_option("-v", "--vehicle", type='string', default=None, help='vehicle type (ArduPlane, ArduCopter or APMrover2)')
-parser.add_option("-f", "--frame", type='string', default=None, help='''set aircraft frame type
-                     for copters can choose +, X, quad or octa
-                     for planes can choose elevon or vtail''')
-parser.add_option("-C", "--sim_vehicle_sh_compatible", action='store_true', default=False, help='be compatible with the way sim_vehicle.sh works; make this the first option')
+parser.add_option("-v", "--vehicle", type='string', default=None, help="vehicle type (ArduPlane, ArduCopter or APMrover2)")
+parser.add_option("-f", "--frame", type='string', default=None, help="set aircraft frame type for copters can choose +, X, quad or octa for planes can choose elevon or vtail")
```
Comment (OXINARF) [inline Tools/autotest/sim_vehicle.py]: I think this was multi-line on purpose, but @peterbarker can comment on that.

File: Tools/autotest/sim_vehicle.py [hunk 3]
```diff
@@ -460,23 +475,23 @@ def options_for_frame(frame, vehicle, opts):
...
         ret["sitl-port"] = True
 
     if opts.model is not None:
         ret["model"] = opts.model
 
     if (ret["model"].find("xplane") != -1 or
-        ret["model"].find("flightaxis") != -1):
-        ret["sitl-port"] = False
+            ret["model"].find("flightaxis") != -1):
+            ret["sitl-port"] = False
```
Comment (OXINARF) [inline Tools/autotest/sim_vehicle.py]: I'm confused here. Why does the instruction inside the if have its indentation increased? The second part of the if I understand (although I would prefer to have the if all in one line), but the instruction inside should have the same indentation as do the previous/next ifs.

File: Tools/autotest/sim_vehicle.py (hunk 2 above)
Comment (khancyr) [inline Tools/autotest/sim_vehicle.py]: yes I see that but it doesn't work... I can revert it

File: Tools/autotest/sim_vehicle.py (hunk 1 above)
Comment (khancyr) [inline Tools/autotest/sim_vehicle.py]: Will correct that

File: Tools/autotest/sim_vehicle.py (hunk 3 above)
Comment (khancyr) [inline Tools/autotest/sim_vehicle.py]: E129 visually indented line with same indent as next logical line

**Comment by khancyr** [comment]
I didn't use https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings . Should we use it ?

**Comment by lucasdemarchi** [comment]
@khancyr ping :)

**Comment by khancyr** [comment]
pong

**Comment by OXINARF** [comment]
@khancyr You haven't addressed one comment I made. If we can't bend the tools to not do that indent automatically then I think we need to revert it manually.

**Comment by khancyr** [comment]
@oxinarf you mean your comment for line 488 ?  I can disable this correction and revert to old intentation

**Comment by khancyr** [comment]
I am not sure for the cygwin_pidof part. Should I correct it or keep it as it ?

**Comment by OXINARF** [comment]
@khancyr Yes, it is in a different line now, but that is the comment. That correction makes no sense to me, every statement inside ifs with same indentation, should have the same indentation too.

Regarding the cygwin_pidoff you already corrected it.

**Comment by khancyr** [comment]
Ok I will recorrect that. 
For cygwin_pidof, I correct it but in previous pr, peterbarker wasn't it was right to do it

**Comment by OXINARF** [comment]
@khancyr Sorry, I was thinking of under_cygwin and not cygwin_pidoff. For cygwin_pidoff, I think the concern @peterbarker raised was that changing the variable names wouldn't keep it consistent with the previous sim_vehicle script, but for me that isn't a problem, I believe we need to move on.

**Comment by peterbarker** [comment]
On Mon, 25 Jul 2016, khancyr wrote:

> Ok I will recorrect that.
> For cygwin_pidof, I correct it but in previous pr, peterbarker wasn't it was
> right to do it

I'll get over it :-)  The good outweighs the bad.

**Comment by lucasdemarchi** [comment]
Applied, thanks!

**Pull request closed without merging**